src/ContentApiBundle.php

```diff
@@ -4,12 +4,19 @@
 
 namespace Libero\ContentApiBundle;
 
+use Libero\ContentApiBundle\DependencyInjection\Compiler\ApiProblemPass;
 use Libero\ContentApiBundle\DependencyInjection\ContentApiExtension;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Extension\ExtensionInterface;
 use Symfony\Component\HttpKernel\Bundle\Bundle;
 
 final class ContentApiBundle extends Bundle
 {
+    public function build(ContainerBuilder $container) : void
+    {
+        $container->addCompilerPass(new ApiProblemPass());
```

Comment:
so bundles can use other bundles

Reply:
ah no, it's just a suggestion but they don't have a direct code dependency, I think